### PR TITLE
fix(version): fix libpq version

### PR DESCRIPTION
**Pull request by thanh-nguyen-dang**
### Bug Fixes
* Fixing `psycopg2.OperationalError: SCRAM authentication requires libpq version 10 or above` for `libpq` to connect to the new version of PostgreSQL, fix from [here](https://stackoverflow.com/a/62808487/1030110).

### CODE REVIEW

**Comment by paulineribeyre** [comment]
in qa-midrc tube fails with `psycopg2.OperationalError: SCRAM authentication requires libpq version 10 or above`, I tried using this branch and got the same error

File: .travis.yml [hunk 1]
```diff
@@ -1,14 +1,16 @@
+dist: bionic
+
 sudo: required
 
 language: python
 
 python:
-  - "3.7"
   - "3.8"
+  - "3.9"
```
Comment (m0nhawk) [inline .travis.yml:9]:
```suggestion
  - "3.9"
  - "3.10"
```

Comment (m0nhawk) [reply .travis.yml:9]: Let's add 3.10 as well.

File: .travis.yml [hunk 2]
```diff
@@ -1,14 +1,16 @@
...
 
 matrix:
  allow_failures:
-   - python: "3.8"
+   - python: "3.9"
```
Comment (m0nhawk) [inline .travis.yml:13]:
```suggestion
   - python: "3.9"
   - python: "3.10"
```

File: dev-requirements.txt [hunk 3]
```diff
@@ -1,8 +1,5 @@
-pytest==3.2.3
-pytest-cov==2.5.1
+pytest==5.4.3
```
Comment (m0nhawk) [inline dev-requirements.txt:1]: `dev-requirements.txt` are not needed for Poetry.

File: requirements.txt [hunk 4]
```diff
@@ -1,16 +1,15 @@
 cdislogging==1.0.0
```
Comment (m0nhawk) [inline requirements.txt:1]: `requirements.txt` is not needed for Poetry.

File: setup.py [hunk 5]
```diff
@@ -9,8 +9,8 @@
         "hdfs==2.1.0",
```
Comment (m0nhawk) [inline setup.py:9]: `setup.py` is not needed for Poetry.

**Review by m0nhawk — APPROVED** [review]

**Pull request merged**